corehq/apps/users/models.py

```diff
@@ -2593,7 +2594,41 @@ def send_request_email(self):
                                     email_from=settings.DEFAULT_FROM_EMAIL)
 
 
-class Invitation(QuickCachedDocumentMixin, Document):
+class SQLInvitation(SyncSQLToCouchMixin, models.Model):
+    email = models.CharField(max_length=255, db_index=True)
+    invited_by = models.CharField(max_length=126)           # couch id of a WebUser
+    invited_on = models.DateTimeField()
+    is_accepted = models.BooleanField(default=False)
+    domain = models.CharField(max_length=255)
+    role = models.CharField(max_length=100, null=True)
+    program = models.CharField(max_length=126, null=True)   # couch id of a Program
+    supply_point = models.CharField(max_length=126, null=True)  # couch id of a Location
+    couch_id = models.CharField(max_length=126, null=True, db_index=True)
+
+    _inviter = None
```

Comment:
what's this?

Reply:
It's from a 2011 commit that references old user ids...maybe there was a time when we changed id schemas? https://github.com/dimagi/commcare-hq/commit/4a10b1643e5481730983cd971f64e5faa88732c5

Added 1d3e0ff822d to the next PR to turn it into a local variable.